### DISCUSSION TITLE
Multi Video Stream

### DISCUSSION THREAD

**Discussion by Kogel9701** [mikel-brostrom/yolo_tracking#571] [Q&A] [answered]
I found that this project can only load one video for processing.
 I want to know if there is an interface that can pass two or more videos at the same time, and deal with them at the same time and display it?

I would appreciate any suggestions to resolve this issue.
Thanks.

**Comment by mikel-brostrom** [comment, marked as answer]
Pass a `.txt` to source containing you video paths

**Reply by Kogel9701** [reply]
That really can pass two or more videos by pass a .txt to source. But it still process video one by one. What I want is the tracking result of the two videos can be displayed at the same time.

**Reply by quintonx1211** [reply]
> That really can pass two or more videos by pass a .txt to source. But it still process video one by one. What I want is the tracking result of the two videos can be displayed at the same time.

Have you solved this?  I have the same demand with you, pls respond if you solved it. TIA.

**Reply by LWQ2EDU** [reply]
I believe this is a multiprocess/thread question. You can use the multiprocessing/threading module to make different sources run in different processes/threads. Such as this [blog](https://blog.gitnote.cn/post/somePyRunTogether/)

**Comment by Minniesse** [comment]
Hi, How can I do this... I use .txt file and it run one by one how can I fix this :)
